src/store/messages/saga.ts

```diff
@@ -277,16 +277,19 @@ export function* stopSyncChannels(action) {
 
 export function* receiveNewMessage(action) {
   const { channelId, message } = action.payload;
+  const channelIdWithoutPrefix = channelId.replace('sendbird_group_channel_', '');
```

Comment:
Can we move the replacement logic to `src/lib/chat/index.ts`? In the coming weeks we have ambitions to replace the message notification event system. It would be nice to isolate our Sendbird specific logic to a single file.